api/conftest.py

```diff
@@ -170,3 +176,41 @@ def master_api_key_client(master_api_key):
     api_client = APIClient()
     api_client.credentials(HTTP_AUTHORIZATION="Api-Key " + master_api_key)
     return api_client
+
+
+@pytest.fixture()
+def test_user(django_user_model):
+    return django_user_model.objects.create(email="user@example.com")
+
+
+@pytest.fixture()
+def test_user_client(api_client, test_user):
```

Comment:
I'm not sure why we've had to create new fixtures here? We already have the `admin_client` fixture don't we?

Reply:
Yeah, but that uses the admin user (admin user have access to everything, right? )

Reply:
No, the admin user is actually org admin
https://github.com/Flagsmith/flagsmith/blob/main/api/conftest.py#L33

Reply:
Ah yes, if you include the organisation (which most fixtures do eventually I suppose). Ok, then can we try and organise the fixtures a bit better at least so it's (hopefully) a bit more obvious why we have 2 client fixtures? i.e. let's put the user fixture next to the admin_user fixture and the client fixture next to the admin_client fixture.